<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>897</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1176</width>
         <height>979</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Máy khắc laser</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Cổng</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cbxLaserPort">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btnEnum">
             <property name="text">
              <string>Liệt kê</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>PLC</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="11" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Stopper và offset vị trí</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spxPLCPort">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="txtRegStop"/>
           </item>
           <item row="6" column="3">
            <widget class="QLineEdit" name="txtRegTriggerConfirm"/>
           </item>
           <item row="9" column="2">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Barcode OK</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txtPLCIP">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="8" column="3">
            <widget class="QLineEdit" name="txtRegBarcodeConfirm"/>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="txtRegTrigger"/>
           </item>
           <item row="8" column="2">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Xác nhận barcode</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Xác nhận trigger</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="txtRegPassMode"/>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="txtRegStopper"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>IP</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Sẵn sàng</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="txtRegBarCode"/>
           </item>
           <item row="9" column="1">
            <widget class="QLineEdit" name="txtRegBarcodeData"/>
           </item>
           <item row="7" column="2">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Xác nhận dừng</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Dữ liệu bar code</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Dừng</string>
             </property>
            </widget>
           </item>
           <item row="9" column="3">
            <widget class="QLineEdit" name="txtRegBarcodeOK"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Cổng</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Passmode</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QLineEdit" name="txtRegStopConfirm"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Barcode</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Stopper</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Trigger</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLineEdit" name="txtRegSWReady"/>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Kết quả in</string>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Barcode NG</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLineEdit" name="txtRegPrintRes"/>
           </item>
           <item row="10" column="3">
            <widget class="QLineEdit" name="txtRegBarcodeNG"/>
           </item>
           <item row="11" column="1" colspan="3">
            <widget class="TableView" name="tbvStoppers">
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>24</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>24</number>
             </attribute>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txtRegCvWidth"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Độ rộng băng tải</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>User</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="2">
            <widget class="QPushButton" name="tbnRemoveUser">
             <property name="text">
              <string>Xóa</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btnAddUser">
             <property name="text">
              <string>Thêm</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="btnResetUser">
             <property name="text">
              <string>Reset mật khẩu</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1" rowspan="4">
            <widget class="TableView" name="tbvUsers">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>22</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>22</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Đường dẫn dữ liệu</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Đường dẫn</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txtDataDir"/>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btnBrowseDataDir">
             <property name="text">
              <string>Chọn</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Bảo vệ</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Mật khẩu admin</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtPassword"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../ui/icons.qrc">
     <normaloff>:/icons/save_48px.png</normaloff>:/icons/save_48px.png</iconset>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../ui/icons.qrc">
     <normaloff>:/icons/reset_48px.png</normaloff>:/icons/reset_48px.png</iconset>
   </property>
   <property name="text">
    <string>Tải lại</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ui/icons.qrc"/>
 </resources>
 <connections/>
</ui>
